### ISSUE TITLE
Introduce Homebrew for Mac OS user

### ISSUE THREAD

**Issue by namkyu1999** [litmuschaos/litmusctl#205]
[The installation guide](https://github.com/litmuschaos/litmusctl?tab=readme-ov-file#installation) is a little bit complicated. Is there any plan to support homebrew package manager?

If we use homebrew, we can install litmusctl simply using below code.

`brew install litmusctl`

**Comment by SarthakJain26** [comment]
Yes, this is already WIP, @deep-poharkar any updates on this? You can raise the PR by implementing what we discussed earlier

**Comment by deep-poharkar** [comment]
hey @SarthakJain26 , as discussed i feel we need to start releasing updates with tags. I will make a draft PR with the formula I have so that we can have an open discussion on it. is it ok?

**Comment by SarthakJain26** [comment]
Sounds great @deep-poharkar 🙌

**Comment by shivam-Purohit** [comment]
https://github.com/Homebrew/homebrew-core/pull/173562
